.pre-commit-config.yaml

```diff
@@ -91,3 +91,11 @@ repos:
       pass_filenames: false
       # inv tidy-all takes several seconds so only run it on pre-push
       stages: [pre-push]
+    - id: shell-set-x
+      name: shell-set-x
```

Comment:
Suggestion. Rename the jobs something like `shell-check-no-set-x` otherwise it seems that the script is setting `set -x` for all the scripts, which is the opposite of what is doing